srcs/runners_manager/runner/Manager.py

```diff
@@ -135,7 +135,7 @@ def manage_runners(self):
                 manager.delete_runner(runner)
 
             # Create if it's still not enough
-            while self.need_new_runner(manager):
+            if self.need_new_runner(manager):
```

Comment:
- [ ] Add log with the number of with their respective status.
```
15 runners online
2 runners offline
1 runner creating
```

Reply:
Agree with Thomas